src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
```

Comment:
Please pay attention to your css properties order
https://9elements.com/css-rule-order/
Please fix all cases